access/validator.go

```diff
@@ -421,7 +450,7 @@ func (v *TransactionValidator) checkSufficientBalanceToPayForTransaction(ctx con
 	}
 
 	// return no error if payer has sufficient balance
-	if bool(canExecuteTransaction) {
+	if canExecuteTransaction {
```

Comment:
I think this was needed because `canExecuteTransaction` is a `cadence.Bool`. is it working this way?